janet/test/gilded-rose.janet

```diff
@@ -0,0 +1,10 @@
+(import ../gilded-rose/main)
+(use judge)
+
+(do
+  (def items 
+    [@{ :name "foo" :quality 3 :sell-in 5}
+     @{ :name "bar" :quality 6 :sell-in 6}])
+  (main/update-quality items)
+  (def found (find |(= ($ :quality) 2) items))
```

Comment:
I think this is an assertion that is also an addition compared to the other languages? I wasn't sure what this line does.